multiqc/modules/qualimap/QM_RNASeq.py

```diff
@@ -153,6 +153,15 @@ def parse_reports(self):
         )
 
     if len(self.qualimap_rnaseq_cov_hist) > 0:
+
+        # Make a normalised percentage version of the coverage data
+        self.qualimap_rnaseq_cov_hist_percent = dict()
+        for s_name in self.qualimap_rnaseq_cov_hist:
+            self.qualimap_rnaseq_cov_hist_percent[s_name] = OrderedDict()
+            total = sum(self.qualimap_rnaseq_cov_hist[s_name].values())
+            for k, v in self.qualimap_rnaseq_cov_hist[s_name].items():
+                self.qualimap_rnaseq_cov_hist_percent[s_name][k] = (v / total) * 100
```

Comment:
Minor: multiplying by `100` can occasionally have a weird effect by making the result an `int` with no decimal places. After hitting this a few times, I have now got into the habit of always multiplying by `100.0` which forces the result to be a `float`.

```suggestion
                self.qualimap_rnaseq_cov_hist_percent[s_name][k] = (v / total) * 100.0
```